src/components/shared/BasePopup/base.popup.module.scss

```diff
@@ -0,0 +1,72 @@
+@import "../../../assets/variables";
+@import "../../../assets/fonts";
+
+.popupWrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  height: 100vh;
+  width: 100%;
+
+  z-index: 99;
```

Comment:
move to variables?

Reply:
-

Reply:
why ?

Reply:
Why move it to variables?

Reply:
I believe it to be useful to store all z-indexes in one place in order not to search in the future "Why my new custom popup is below this one"

Reply:
Show me an example

Reply:
I want to create some toast notification that should be displayed above this popup. Why should I search for this popup component and its styles file in order to find its z-index, instead of just look at the varaibles file and create a new variable with greater z-index or just use the same z-index as a variable.

Reply:
How are you going to find it in variables?

And why not to check for z-index in chrome console (two clicks), and add a higher index.

Reply:
Moved to variables